09-redux/classworks/Malkov_Aleksei/src/pages/home/Home.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { withRedditApi } from 'hooks/useRedditApi';
+import Header from 'components/header';
+import Title from 'components/title';
+import PostBlock from 'components/post_block';
+import styles from './home.module.css';
+import Spinner from 'components/spinner';
+
+function Home(props){
+  const { fetchReddit } = props;
+  const [reactSubreddit, setReactSubreddit] = props.reactSubreddit;
+  const [about, setAbout] = props.about;
+
+  useEffect(() => {
+    const preload = async () => {
+      const data = await fetchReddit('/r/Twitch/hot').then(res => res.json());
```

Comment:
no fetching allowed in components. all fetches should be in actions/redux. we only dispatch them in components